src/rf/apps/core/models.py

```diff
@@ -133,6 +133,16 @@ class Meta:
     thumb_large_key = CharField(max_length=255, blank=True, default='',
                                 help_text='S3 key for large thumbnail')
 
+    def retry_possible(self):
+        """
+        Returns true if the layer failed during the EMR steps.
+        """
+        thumbnailed_images = self.layer_images.filter(
+            status=enums.STATUS_THUMBNAILED)
+        all_thumbnailed = self.layer_images.count() == \
+            thumbnailed_images.count()
+        return self.status == enums.STATUS_FAILED and all_thumbnailed
```

Comment:
If the images are thumbnailed, but the layer failed, then it follows that it failed during the EMR steps. But that's going change soon with various changes that people are making, so maybe you should hold off on reviewing this PR for now.